platforms/forge/build.gradle

```diff
@@ -33,9 +34,27 @@ minecraft {
     // makeObfSourceJar = false // an Srg named sources jar is made by default. uncomment this to disable.
 }
 
+repositories
+{
+    maven 
+    {   // Worldedit API
+   		url "https://maven.sk89q.com/repo/" 
+   	}
+}
+
 dependencies
 {
     compile project(':common')	
+	provided ('com.sk89q.worldedit:worldedit-forge-mc1.11:6.1.6') {
```

Comment:
Would be interested to know what happens if you import and use WE classes at mod init, and then run the client without WE. It should crash with something like a NoClassDefFoundError / ClassNotFoundException / MethodNotFoundException. It think it doesn't currently crash because the OTG classes that use the WE dependencies are never loaded if WE isnt installed, OTG commands never reach that code, which is nice.